### PR TITLE
add upcoming based on steering feedback

### PR DESCRIPTION
**Pull request by linsun**
will also cherry pick this to istio.io.  

[ ] Configuration Infrastructure
[x] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by linsun** [comment]
cc @craigbox @christian-posta @ericvn

File: content/en/blog/2021/discovery-selectors/index.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Use discovery selectors to configure namespaces for your Istio service mesh"
+title: "Upcoming in Istio 1.10 - use discovery selectors to configure namespaces for your Istio service mesh"
```
Comment (craigbox) [inline content/en/blog/2021/discovery-selectors/index.md:2]: I would leave this as is in the title...

**Review by craigbox — CHANGES_REQUESTED** [review]

**Comment by linsun** [comment]
@craigbox thanks, your feedback is resolved. PTAL! :)

**Review by craigbox — APPROVED** [review]
LGTM, please commit same file/s to 1.9 and/or cherrypick!

**Pull request merged**